### PR TITLE
Product Controller, DTO, Repository

### PR DESCRIPTION
**Pull request by josephum**

### CODE REVIEW

File: src/main/java/com/example/accountingapp/service/ProductService.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package com.example.accountingapp.service;
+
+import com.example.accountingapp.dto.ProductDTO;
+
+import java.util.List;
+
+public interface ProductService {
```
Comment (AndreSelv) [inline src/main/java/com/example/accountingapp/service/ProductService.java:7]: Compiler error... we have to add serviceImpl class to run it properly

File: src/main/java/com/example/accountingapp/controller/ProductController.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+//            return "/product/product-edit";
+//        }
+        System.out.println("To update id: "+id);
+//        // productService.update(productDTO);
+        return "redirect:/product/list";
+    }
+
+    @GetMapping("/delete/{id}")
+    public String deleteClient(@PathVariable("id") Long id) {
+        System.out.println("To delete id: "+id);
```
Comment (AndreSelv) [inline src/main/java/com/example/accountingapp/controller/ProductController.java]: Remove unnecessary statement "sout"

**Review by AndreSelv — CHANGES_REQUESTED** [review]
Compiler error... we have to add serviceImpl class to run it properly

File: src/main/java/com/example/accountingapp/service/ProductService.java (hunk 1 above)
Comment (josephum) [reply src/main/java/com/example/accountingapp/service/ProductService.java:7]: Please try now!

File: src/main/java/com/example/accountingapp/controller/ProductController.java (hunk 2 above)
Comment (josephum) [reply src/main/java/com/example/accountingapp/controller/ProductController.java]: Done

**Review by MehmetKaldirim — APPROVED** [review]
list work properly,good job...

**Review by AndreSelv — APPROVED** [review]
Looks good for me! Great job.

**Pull request merged**